security/cmd/node_agent_k8s/main.go

```diff
@@ -137,17 +149,30 @@ func init() {
 		pns = strings.Split(pluginNames, ",")
 	}
 
+	enableWorkloadSdsEnv := true
+	val := os.Getenv(enableWorkloadSDS)
+	if env, err := strconv.ParseBool(val); err == nil {
+		enableWorkloadSdsEnv = env
+	}
+	enableIngressGatewaySdsEnv := false
+	val = os.Getenv(enableIngressGatewaySDS)
+	if env, err := strconv.ParseBool(val); err == nil {
+		enableIngressGatewaySdsEnv = env
+	}
```

Comment:
what is the need for this feature flag code when you are simply using it to set the default value of a cli option? I suggest you remove this. And tweak the helm charts to turn the flag on for ingress gateways.